.gitignore

```diff
@@ -0,0 +1,3 @@
+.benchdir
```

Comment:
Please add ticket id's to commit messages.

I run `./ttbench install` and then `./ttbench run -b ycsb` and got this output:

```
[s.bronnikov@tarantool-core-dev-mcs1 bench-run]$ ./ttbench run -b ycsb
[INFO] ---
[INFO] TPCH_SKIP_SQLITE: '1'
[INFO] NOSQLBENCH_TIMELIMIT: '20'
[INFO] TPCC_TIME: '20'
[INFO] YCSB_RECORDCOUNT: '1000'
[INFO] TARANTOOLCTL_EXECUTABLE: tarantoolctl
[INFO] TPCC_WARMUPTIME: '5'
[INFO] YCSB_OPERATIONCOUNT: '1000'
[INFO] SYSBENCH_TESTS: all
[INFO] NOSQLBENCH_WORKLOADSIZE: '100000'
[INFO] TARANTOOL_EXECUTABLE: tarantool
[INFO] NOSQLBENCH_RPS: '20000'
[INFO] LINKBENCH_REQUESTERS: '1'
[INFO] LINKBENCH_REQUESTS: '1000'
[INFO] YCSB_RUNS: '1'
[INFO] TPCC_WAREHOUSES: '15'
[INFO] SYSBENCH_TIME: '5'
[INFO] LINKBENCH_WORKLOAD_SIZE: '1000'
[INFO] CBENCH_VINYL_WORKLOAD: '50'
[INFO] CBENCH_MEMTX_WORKLOAD: '10000'
[INFO] NOSQLBENCH_BATCHCOUNT: '10'
[INFO] SYSBENCH_RUNS: '1'
[INFO] ...
[INFO] 
=============== a
---------------- a: 1
/usr/bin/env: 'python': No such file or directory
sync passed
disk cache was not dropped
/usr/bin/env: 'python': No such file or directory
[ERROR] script='cd /home/s.bronnikov/work/bench-run/.benchdir/ycsb && /home/s.bronnikov/work/bench-run/.benchdir/ycsb/run.sh' failed with ec='1'
[ERROR] userdata: 0x41d81b68
[s.bronnikov@tarantool-core-dev-mcs1 bench-run]$ 
```

Why "disk cache was not dropped"?

Python interpreter was not found. Why?